MultiServer.py

```diff
@@ -1474,6 +1475,14 @@ async def process_client_cmd(ctx: Context, client: Client, args: dict):
             package["games"] = games
             await ctx.send_msgs(client, [{"cmd": "DataPackage",
                                           "data": package}])
+        elif "games" in args:
```

Comment:
Think it would be better to
1. prefer the new version (`if "games" in args and args["games"]: new_behaviour; else: old_behaviour`)
2. merge the code in a way that `args["games"]` simply overwrites `games` and everything else stays the same